### PR TITLE
Swapped all limb armor from strapped to outer

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
Balance "Moved all limb strapped armor to the outer layer"

#### Purpose of change
completes a large chunk of #52514 

#### Describe the solution
really simple find replace of all "BELTED" tags in arms_armor and legs_armor with "OUTER"

Some other stuff needs to be moved as well but this is the largest portion of the items that need moved. These items being moved also helps to showcase the power of #52468 . 

#### Describe alternatives you've considered
Stuffing everything together in a monolithic PR

#### Testing
Load into game. 
Didn't get any errors.
Spawned some metal leg guards they are now on outer.

### CODE REVIEW

**Comment by Photoloss** [comment]
Even with sub-part locations this does not categorically make sense. Something like elbow pads can in fact be readily strapped and worn above soft textiles which would go in the "Outer" layer such as a thick winter jacket, adding them to a rigid armour shell would be problematic of course.

There's also the option of wearing certain currently "Strapped" items _below_ the "Outer" or even "inner" (main clothing) layer as proposed in the linked issue. Semi-rigid sparring gear such as [these](https://www.flipkart.com/gautam-sports-martial-arts-arm-guard-mma/p/itmb8168b79d53b2) can be worn within the wide sleeves of a gi (and by extension most coats and jackets or even a decently slack wool jumper) and cause no more obstruction than they already do anyway. The key here is that these items only cover rigid areas of the limbs rather than the joints, and gi sleeves do not reach the hand joint to get tangled up in any meaningful way (this might be a concern for the winter coat and wool jumper). A well-fitted hard shell would work just as well, but would trade lower core thickness for less coverage as the edges need to be rounded off, padded and kept away from joints.

Ultimately the key point is that "Strapped" items are adjustable to different attachment configurations while something like a sleeve or an articulated armour joint is not. If a nerf is desired for gameplay reasons these items should be split into "tube" and "plate" configurations with the strapped plates having roughly half the coverage: that sparring gear covers maybe 30% of your arm in meaningful padding, 50% max if you count the cloth, not the 80% "hard arm guards" currently have. No way elbow pads absorb 30% of random arm hits either. Additionally straps can come loose quite easily (this definitely also happens with said sparring gear) which for annoyance reduction would reduce the coverage to almost 0 until reattached instead of having the item actually fall off the player.

**Comment by bombasticSlacks** [comment]
you're balance suggestions are being addressed in other PR's. This isn't about nerfs it's about consistency. Currently things like plate arms and legs are strapped while the body armor is outer and then when combined into a single suit they are all outer. It makes no sense.

**Comment by Photoloss** [comment]
> you're balance suggestions are being addressed in other PR's. This isn't about nerfs it's about consistency. Currently things like plate arms and legs are strapped while the body armor is outer and then when combined into a single suit they are all outer. It makes no sense.

And for plate armour I mostly agree, there are only very specific configurations which could possibly work as "Strapped" under my definition. Mainly half-shell forearm/shin protectors.

But the changed items also include *knee pads* and [archery vambraces](https://www.outfit4events.com/eur/product/64-leather-archery-vambrace/). Those have enough slack and flexibility to genuinely strap on top of pretty much everything that is not a hard plate armour, so the necessary change is for rigid clothing/armour pieces to block certain other items or to remove same-(sub)limb encumbrance penalties for multiple different flexible items in the same layer. I can wear two layers of those archery vambraces, I definitely can not wear both plate and wood armguards regardless of how few textiles are underneath, and combining plate armguards with the sparring gear would strain the straps a lot if it works at all.

TL;DR: exclude elbow/knee pads and soft fabric/leather items from this change, but block them from being worn with plate armour regardless of layer. Not for balance, but for the realism point of plate armour being *rigid* and thus physically obstructing you from wearing stuff.

**Comment by bombasticSlacks** [comment]
I'm just implementing I don't get final say in any of this however: 

You could also strap metal arm guards or leather vambraces on top of a jacket or plate just as easily as elbow pads/knee pads it would just be uncomfortable and unweildly because of the thickness and balance. The "slack and flexibility" of elbow / knee pads is already represented by them being 0 encumbrance. The other things you are describing seem like they should be normal layer not strapped. however the current leather vambraces in game are 3mm thick which is twice the thickness of a leather coat. If someone added thinner vambraces they could go on normal or inner like the cut protection sleeves or argue for the move as is in a separate PR.

**Comment by Photoloss** [comment]
> I'm just implementing I don't get final say in any of this however:
> 
> You could also strap metal arm guards or leather vambraces on top of a jacket or plate just as easily as elbow pads/knee pads it would just be uncomfortable and unweildly because of the thickness and balance. The "slack and flexibility" of elbow / knee pads is already represented by them being 0 encumbrance. The other things you are describing seem like they should be normal layer not strapped. however the current leather vambraces in game are 3mm thick which is twice the thickness of a leather coat. If someone added thinner vambraces they could go on normal or inner like the cut protection sleeves or argue for the move as is in a separate PR.

You seem to be misunderstanding. A metal half-shell has a roughly semicircular cross-section with a fixed radius. If you try to wear two of those the outer one will be offset significantly, no longer conforming to the shape of the arm and rattling quite a bit. This is not just an encumbrance penalty, it flat out no longer works as stable limb armour for e.g. blocking a sword cut. The inner one still does. Meanwhile even at 3mm thick I can bend the vambraces and tie the fastening cord on the second outer pair. It will have very marginally lower coverage (larger gap in the circumference) and it will be very uncomfortable (I never said to remove same-item-shape stacking penalties), but it does work as an effective doubling of the armour thickness.

This is separate from the base 0 encumberance. Vambraces and shin guards should be close to 0 because they do not interact with joints, elbow pads probably above 0 base because they do. But that is for the item by itself, my point is based entirely on the fact that genuinely strap-tied items do not cause *additional* interference when worn above most soft textiles. Which is the part this PR breaks. You should be able to wear elbow pads and a jacket without an additional penalty, but not metal arm guards with that same jacket. They physically do not function the same so homogenisation is not desirable.
